node_modules/oae-core/about/about.html

```diff
@@ -0,0 +1,48 @@
+<!-- CSS -->
```

Comment:
The `about` widget should be renamed to `aboutgroup`, given that it will only work for groups as is